[rospy.client][INFO] 2017-10-18 07:50:21,155: init_node, name[/state_driving_away], pid[5123]
[xmlrpc][INFO] 2017-10-18 07:50:21,156: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2017-10-18 07:50:21,158: ROS Slave URI: [http://10.0.0.116:42757/]
[xmlrpc][INFO] 2017-10-18 07:50:21,159: Started XML-RPC server [http://10.0.0.116:42757/]
[rospy.impl.masterslave][INFO] 2017-10-18 07:50:21,159: _ready: http://10.0.0.116:42757/
[xmlrpc][INFO] 2017-10-18 07:50:21,160: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-10-18 07:50:21,162: Registering with master node http://10.0.0.116:11311
[rospy.init][INFO] 2017-10-18 07:50:21,259: registered with master
[rospy.rosout][INFO] 2017-10-18 07:50:21,260: initializing /rosout core topic
[rospy.rosout][INFO] 2017-10-18 07:50:21,276: connected to core topic /rosout
[rospy.simtime][INFO] 2017-10-18 07:50:21,285: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-10-18 07:50:21,378: topic[/state_command] adding connection to [/state_follow_waypoint], count 0
[rospy.internal][INFO] 2017-10-18 07:50:21,381: topic[/state_command] adding connection to [/state_driving_to_cone], count 1
[rospy.internal][INFO] 2017-10-18 07:50:21,404: topic[/state_command] adding connection to [http://10.0.0.116:40494/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:21,405: topic[/state_command] adding connection to [http://10.0.0.116:38297/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:21,426: topic[/state_command] adding connection to [http://10.0.0.116:42757/], count 2
[rospy.internal][INFO] 2017-10-18 07:50:21,432: topic[/state_command] adding connection to [/state_driving_away], count 2
[rospy.internal][INFO] 2017-10-18 07:50:21,538: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-10-18 07:50:21,627: topic[/state_command] adding connection to [/state_avoiding_obstacle], count 3
[rospy.internal][INFO] 2017-10-18 07:50:21,690: topic[/state_command] adding connection to [http://10.0.0.116:38324/], count 3
[rospy.internal][INFO] 2017-10-18 07:50:22,201: topic[/mavros/setpoint_raw/target_local] adding connection to [http://10.0.0.116:45828/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:22,262: topic[/mavros/state] adding connection to [http://10.0.0.116:45828/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:22,312: topic[/mavros/battery] adding connection to [http://10.0.0.116:45828/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:22,316: topic[/mavlink/from] adding connection to [http://10.0.0.116:45828/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:22,320: topic[/mavros/local_position/pose] adding connection to [http://10.0.0.116:45828/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:22,569: topic[/mavros/mission/waypoints] adding connection to [http://10.0.0.116:45828/], count 0
[rospy.internal][INFO] 2017-10-18 07:50:22,588: topic[/state_command] adding connection to [http://10.0.0.116:42545/], count 4
[rospy.internal][INFO] 2017-10-18 07:50:22,617: topic[/exec_command] adding connection to [/exec], count 0
[rospy.internal][INFO] 2017-10-18 07:50:22,693: topic[/mavros/rc/override] adding connection to [/mavros], count 0
[rospy.core][INFO] 2017-10-18 07:51:05,732: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-10-18 07:51:06,034: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-10-18 07:51:06,037: topic[/mavros/rc/override] removing connection to /mavros
[rospy.internal][INFO] 2017-10-18 07:51:06,041: topic[/state_command] removing connection to /state_follow_waypoint
[rospy.internal][INFO] 2017-10-18 07:51:06,042: topic[/state_command] removing connection to /state_driving_to_cone
[rospy.internal][INFO] 2017-10-18 07:51:06,043: topic[/state_command] removing connection to /state_driving_away
[rospy.internal][INFO] 2017-10-18 07:51:06,045: topic[/state_command] removing connection to /state_avoiding_obstacle
[rospy.internal][INFO] 2017-10-18 07:51:06,046: topic[/exec_command] removing connection to /exec
[rospy.internal][WARNING] 2017-10-18 07:51:06,054: Unknown error initiating TCP/IP socket to 10.0.0.116:59484 (http://10.0.0.116:45828/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][INFO] 2017-10-18 07:51:06,055: topic[/mavros/mission/waypoints] removing connection to http://10.0.0.116:45828/
[rospy.internal][ERROR] 2017-10-18 07:51:06,056: Unable to initiate TCP/IP socket to 10.0.0.116:42659 (http://10.0.0.116:42757/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2017-10-18 07:51:06,057: topic[/mavros/mission/waypoints] removing connection to http://10.0.0.116:45828/
[rospy.internal][INFO] 2017-10-18 07:51:06,058: topic[/mavros/state] removing connection to http://10.0.0.116:45828/
[rospy.internal][INFO] 2017-10-18 07:51:06,059: topic[/mavlink/from] removing connection to http://10.0.0.116:45828/
[rospy.internal][INFO] 2017-10-18 07:51:06,062: topic[/mavros/local_position/pose] removing connection to http://10.0.0.116:45828/
[rospy.internal][INFO] 2017-10-18 07:51:06,068: topic[/mavros/battery] removing connection to http://10.0.0.116:45828/
[rospy.internal][WARNING] 2017-10-18 07:51:06,069: Unknown error initiating TCP/IP socket to 10.0.0.116:59484 (http://10.0.0.116:45828/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.internal][WARNING] 2017-10-18 07:51:06,064: Unknown error initiating TCP/IP socket to 10.0.0.116:59484 (http://10.0.0.116:45828/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.internal][INFO] 2017-10-18 07:51:06,070: topic[/state_command] removing connection to http://10.0.0.116:40494/
[rospy.internal][INFO] 2017-10-18 07:51:06,072: topic[/state_command] removing connection to http://10.0.0.116:38297/
[rospy.internal][INFO] 2017-10-18 07:51:06,072: topic[/state_command] removing connection to http://10.0.0.116:42757/
[rospy.internal][INFO] 2017-10-18 07:51:06,073: topic[/state_command] removing connection to http://10.0.0.116:38324/
[rospy.internal][INFO] 2017-10-18 07:51:06,074: topic[/state_command] removing connection to http://10.0.0.116:42545/
[rospy.internal][WARNING] 2017-10-18 07:51:06,078: Unknown error initiating TCP/IP socket to 10.0.0.116:59484 (http://10.0.0.116:45828/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.internal][INFO] 2017-10-18 07:51:06,076: topic[/mavros/setpoint_raw/target_local] removing connection to http://10.0.0.116:45828/
[rospy.impl.masterslave][INFO] 2017-10-18 07:51:06,079: signal-2
[rospy.core][INFO] 2017-10-18 07:51:06,398: signal_shutdown [atexit]
